projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor.component.ts

```diff
@@ -31,6 +32,8 @@ export class JsonEditorComponent implements OnChanges {
   @Input()
   typeCheckOverrides?: any;
 
+  @Input() customAjv: Ajv.Ajv;
```

Comment:
Would prefer this to be a method, so it doesn't add a dependecy on Ajv. Allows users to use different schema validators.